backend/gen-test-data.sql

```diff
@@ -1,9 +1,186 @@
+
+
+## plants for user1
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (1, true, 'Storczyk z biurka', 'Storczyk na biurku se stoi',
+        'https://zielonyogrodek.pl/i/images/4/9/8/d2FjPTkwMHgxLjQwMA==_src_34498-storczyk_z_supermarketu_fot._RainerBerns_-_Pixabay.com.jpg'
+           , 1, 1);
+
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (1, false, 'Magnolia Ewa', 'Taka stokowa zwykla magnolia, imieniem Ewa, herbu doniczka',
+        'https://cdn1.hk-green.pl/sztuczna-magnolia-w-doniczce-21-cm-p3873329.jpg'
+           , 5, 1);
+
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (3, true, 'Kaktusek', 'Nie chce tanczyc',
+        'https://sztuczne-rosliny.pl/wp-content/uploads/2017/01/sztuczny-kaktus-karnegia-800-800-1.jpg'
+           , 2, 1);
+
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (2, false, 'Fiolek', 'Just fiolek, nic specjalnego w tym fiolku.',
+        'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTBeW_ppritYkcCOgAam8WNcHAwJRO2a57uug&usqp=CAU'
+           , 3, 1);
+
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (2, false, 'Moja paprotka', 'specjalna paprotka, tylko moja',
+        'https://thumbs.img-sprzedajemy.pl/1000x901c/23/ac/94/paprotka-eukaliptus-trawa-trawka-pozostale-kozieglowy-530579814.jpg'
+           , 8, 1);
+
+## plants for user 2
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (1, true, 'Rozyczka z koszyczka', 'roza ze smietnika, moze wstanie',
+           'https://i.pinimg.com/originals/90/b1/1e/90b11e153d00cc913c3f133f043a0b28.jpg'
+        , 4, 2);
+
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (1, false, 'Swieza Stokrotka', 'stokrotka, tak jak ten akademik ds8',
+        'https://www.weranda.pl/data/articles/stokrotka_(8).jpg'
+           , 6, 2);
+
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (3, false, 'moj wyjatkowy kaktus', 'bardzo bardzo specjalny kaktus wygladajacy jak ludzik',
+        'https://succulentcity.com/wp-content/uploads/2019/10/What-Is-Special-About-A-Cactus-4.-Their-unique-growth-habits-sc.jpg'
+           , 2, 2);
+
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (2, false, 'Fiolek', 'Just fiolek, nic specjalnego w tym fiolku.',
+        'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTBeW_ppritYkcCOgAam8WNcHAwJRO2a57uug&usqp=CAU'
+           , 3, 2);
+
+INSERT INTO plants (actual_insolation, favourite, name, note, url, species_id, user_id)
+VALUES (2, true, 'Magnolia Stefan', 'Magnolia, nie reaguje na imie',
+        'https://mardecor.pl/images/KWIATY/YL14-1939.jpg'
+           , 10, 2);
```

Comment:
Jak masz pola związane z enumami np. `needed_insolation`, to indeksują się one od `0`, a nie od `1`, przez to backend się wywala robiąc select'a, bo przy liczbie 3 masz błąd `IndexOutOfRange`